src/components/ebay-calendar/index.marko

```diff
@@ -0,0 +1,110 @@
+// @ts-check
+import {toISO} from "./component"
+
+$ const numMonths = input.numMonths || 1;
+$ const monthDates = [...Array(numMonths)].map((_, i) => component.getMonthDate(state.offset + i))
+
+<div.calendar>
+    <if(input.navigable)>
+        <div.calendar__header>
+            <div.calendar__header--inner>
+                <ebay-icon-button
+                    transparent size="small"
```

Comment:
This should be ideally be in different lines. I misreading it as incorrect param. 
```
<ebay-icon-button
  transparent
  size="small"
  ......
/>
```